From: 13832136087 <ningxiyuyi@126.com>
Date: Thu, 27 Aug 2026 12:53:53 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (210 of 210 strings)

Translation: openkylin-huanghe/kylin-os-manager-device-controller
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-os-manager-device-controller/ko_KP/
---
 .../translations/kylin-os-manager-device-controller_ko_KP.ts            | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/device-controller/translations/kylin-os-manager-device-controller_ko_KP.ts b/plugins/device-controller/translations/kylin-os-manager-device-controller_ko_KP.ts
index 43ce39e..3742f5e 100644
--- a/plugins/device-controller/translations/kylin-os-manager-device-controller_ko_KP.ts
+++ b/plugins/device-controller/translations/kylin-os-manager-device-controller_ko_KP.ts
@@ -661,7 +661,7 @@
         <location filename="../src/database/drivermanagerdatabase.cpp" line="131"/>
         <location filename="../src/database/drivermanagerdatabase.cpp" line="151"/>
         <source>Database Error</source>
-        <translation>자료기지 오류</translation>
+        <translation>데이터베이스 오류</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="18"/>
